packages/sources/wootrade/src/config.ts

```diff
@@ -0,0 +1,16 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'WOOTRADE'
+
+export const DEFAULT_ENDPOINT = 'crypto'
+export const DEFAULT_BASE_URL = 'https://api.staging.woo.network'
```

Comment:
I think we probably should be using the prod endpoint here :sweat_smile: 

```suggestion
export const DEFAULT_BASE_URL = 'https://api.woo.network'
```

Reply:
fixed